### PR TITLE
Better .gitignore

### PR DESCRIPTION
**Pull request by ddekany**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
-/bin/
+build/
+.out/
+bin/
+.bin/
+target/
+
+/gradle.properties
+/archive/
+/META-INF
```
Comment (angelozerr) [inline .gitignore:9]: We need to keep it for MANIFEST.MF

Comment (ddekany) [reply .gitignore:9]: It starts with `/`, so it only ignores the `META-INF` in the repository root.

Comment (angelozerr) [reply .gitignore:9]: Good catch! Ok thanks for clarification

**Comment by angelozerr** [comment]
Thanks!

**Pull request merged**